darts/models/filtering/kalman_filter.py

```diff
@@ -204,48 +280,50 @@ def filter(
         y_values = series.values(copy=False)
         if self._expect_covariates:
             u_values = covariates.values(copy=False)
+
+            # set control signal to 0 if it contains NaNs:
+            # u_values = np.nan_to_num(u_values, copy=True, nan=0.0)
```

Comment:
This should probably be uncommented? Also, it would be good to add tests for missing values both in observations and covariates